test/de/opendiabetes/tests/plugin/importer/MedtronicImporterTest.java

```diff
@@ -124,17 +125,28 @@ public void printLogOnLoadConfiguration() {
         config.setProperty("delimiter", "not a valid delimiter");
         Assert.assertFalse(medtronicImporter.loadConfiguration(config));
 
-        //not providing a value for delimiter
-        config = new Properties();
-        config.setProperty("another property", "value");
-        Assert.assertFalse(medtronicImporter.loadConfiguration(config));
-
 
         config.setProperty("delimiter", "");
         Assert.assertFalse(medtronicImporter.loadConfiguration(config));
 
     }
 
+
```

Comment:
Es sind zwar nur Tests, jedoch wär es schön, wenn auch hier vollständige Sätze stehen würde, also Start mit einem Großbuchstaben und Ende mit einem Punkt

- testdata which cannot 
- Der 2. Satz ist auch irgendwie grammatikalisch komisch :D Wie wär's mit "It also sometimes happens, that you need to delete the lib folder from the plugin in order to get this working."?